src/Survey/ItechArt.Survey.Foundation/Authentication/Validation/Abstractions/IUserValidator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ItechArt.Common.Validation;
+using ItechArt.Common.Validation.Abstractions;
+using ItechArt.Survey.DomainModel;
+using ItechArt.Survey.Foundation.Authentication.Abstractions;
+
+namespace ItechArt.Survey.Foundation.Authentication.Validation.Abstractions;
+
+public interface IUserValidator : IValidator<User, UserRegistrationErrors>
+{
+    ValidationResult<UserRegistrationErrors> ValidatePassword(string password);
```

Comment:
```suggestion
    ValidationResult<UserRegistrationErrors> Validate(User user, string password);
```

Reply:
1. adjust implementation to call base validation for user (before password validation)
2. adjust client to call only this method